src/screens/Distribution/RingChart.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+
+import React, { PureComponent } from "react";
+import { View } from "react-native";
+import * as d3shape from "d3-shape";
+import Svg, { Path, G } from "react-native-svg";
+import colors from "../../colors";
+import type { DistributionItem } from "./DistributionCard";
+
+type Props = {
+  data: *,
+  style?: *,
+};
+
+// TODO refactor this so it's reusable not only with asset distributions
+class RingChart extends PureComponent<Props> {
+  static defaultProps = {
+    data: [],
+  };
+  arcGenerator = d3shape.arc();
+  reducer = (data: *, item: DistributionItem) => {
+    const increment = item.distribution * 2 * Math.PI;
+    const done = data.done;
+
+    if (done) return data;
```

Comment:
I don't see where done is used. sounds like it can be removed?

Reply:
it's a vestigial code left from when I was grouping (remember the red portion that grouped less than 10% assets?) Removed